packages/contracts/migrations/index.ts

```diff
@@ -1,20 +0,0 @@
-import { Deployer } from '@0xproject/deployer';
```

Comment:
Isn't this file still useful? For manual test deployments onto testrpc (portal etc)

Reply:
Yep agreed. Gonna move migrations into a separate package since it's used by 0x.js tests and independently when deploying to mainnet/testnets.